xml file example doesn't work

sevelho opened the pull request:
Attempting to create the VNET using the exemple XML we get a XML validation error
"BadRequest: XML Schema validation error in network configuration ..."

azurecla commented:
Hi __@yamatoya__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <p>The agreement was validated by Microsoft and real humans are currently evaluating your PR.</p>

TTYL, AZPRBOT;

rmca14 commented:
Sent to Steve for review.

stevestein commented:
Hi Takahito,

Where it says: 'If the DTU cap per database is set to 50 DTUs, then it is reasonable to overcommit the group by 5x and set the DTU guarantee for the group to 1,000 DTUs.'
It is actually saying (trying to say!) that it is reasonable to overcommit the group or 'PoolDTUs' by 5x (200*5=1000)

Thanks for pointing this out because it is definitely not clear. I will update the article to clarify.

Thanks again,
Steve

(Ross, please close this pull request as I will update the article based on Takahito's feedback within the next day or 2)

rmca14 commented:
PR closed (test message).

sevelho committed:
xml file example doesn't work

Attempting to create the VNET using the exemple XML we get a XML validation error
"BadRequest: XML Schema validation error in network configuration ..."
